sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/internal/TestInputFileBuilder.java

```diff
@@ -74,7 +76,11 @@ public static int nextBatchId() {
   }
 
   public TestInputFileBuilder setModuleBaseDir(Path moduleBaseDir) {
-    this.moduleBaseDir = moduleBaseDir.normalize();
+    try {
+      this.moduleBaseDir = moduleBaseDir.normalize().toRealPath(new LinkOption[] {LinkOption.NOFOLLOW_LINKS});
+    } catch (IOException e) {
+      this.moduleBaseDir = moduleBaseDir.normalize();
```

Comment:
While this could help for windows path, I don't see a test for it and I am unsure about how this would behave on macOs